### ISSUE TITLE
Formatter breaking Blade comments inside components

### ISSUE THREAD

**Issue by ralphjsmit** [Stillat/blade-parser-typescript#71]
I also noticed that the formatter added spaces between the `{{` and the `--` part of the `{{--` Blade comment syntax when a comment is added inside an HTML tag/Blade component. 

This also caused a server error to break the part of the code:

<img width="1387" alt="Comments" src="https://github.com/Stillat/blade-parser-typescript/assets/59207045/43e8f9e4-e3b4-405c-aa0b-068e2a37df16">

Hereby the original code:

```blade
<x-filament::button
    outlined
    color="secondary"
    {{-- Do not include statePath as parameter to close-modal, otherwise the media picker will update the selected values --}}
    x-on:click="$dispatch('close-modal', {id: 'media-library-picker'})"
>
    Text
</x-filament::button>
```

Thanks!

**Comment by JohnathonKoster** [comment]
Resolved in `1.6.9` 🙂

**Comment by ralphjsmit** [comment]
Thanks!
